WeWatch/WeWatch/Custom TabBar/TabBar.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  TabBar.swift
+//  WeWatch
+//
+//  Created by Anton on 30/12/2024.
+//
+
+import SwiftUI
+
+private struct TabBar: View {
+    
+    @StateObject private var viewModel: TabBarViewModel = .init()
+
+    internal var body: some View {
+        ZStack {
+            viewModel.selectedTab.activeView()
+            CustomTabBar(viewModel: viewModel)
```

Comment:
The `TabBar` should take a `Binding` as an argument instead of relying on the `viewModel` being passed in.